frontend/app/controllers/create-badges.js

```diff
@@ -34,6 +38,25 @@ export default Controller.extend({
           _this.set('userError', userErrors);
         }
       });
+
+      let fontColorCode = '#' + _this.defFontColor;
+      let badgeRecord = _this.get('store').createRecord('badge', {
+        csv        : _this.csvFile,
+        image      : _this.custImgFile,
+        font_color : fontColorCode,
+        uid        : _this.uid,
+        badge_size : 'A3',
+        font_size  : '20',
+        font_type  : 'sans'
+      });
+
+      badgeRecord.save()
+        .then(record => {
+          console.log(record.filename);
```

Comment:
I guess these were for debugging, can be removed now.

Reply:
Sure :+1: